news/2023/2023-08-28-medal-hunter-titles-second-coming.md

```diff
@@ -0,0 +1,70 @@
+
+In order to obtain such a desirable title, you had to be the first player to find the solution for the **Internment** medal and unlock it. **Internment** is a Hush-Hush medal added in the last batch, being so hint-based that developers found unlocking it worth a title. Speaking of hint and sub-hint (referenced as flairs by Ephemeral), they are "Stained in the hues of madness, they rage within a four-walled jail." and "Bask in the shade of an ailing mind." respectively.
+And while only one player could get a shiny "**Medal Hunter**" text under their username, many still have decided to team up with others on the [Medal Hunters Discord server](https://discord.gg/8qpNTs6)!
+
+But before we get to community's attempts to solve it, here are some details they knew from the beginning: 
+The medal is only visible in osu!standard profile, so it's std-only, meaning you can't get it in other gamemodes; 
+If the medal is map-specific, meaning it can only be done on specific mapset(s) - it has to be Featured Artist song; 
+The medal is in a Hush-Hush (Expert) category, which means no [difficulty reduction mods](/wiki/Gameplay/Game_modifier#difficulty-reduction) (or [special mods](/wiki/Gameplay/Game_modifier#special)) are allowed.
+
+Within the first few minutes of the medal releasing, players immediately noticed both hints and especially the medal icon referenced theyaremanycolors. Ever since the beginning of this medal's release, medal hunters knew the medal involved that map. From there, medal hunters were very quick to piece together that the hues from medal hint/flair are hinting at RGB to some capacity, but nobody knew in what way, and players were less focused on the subhint which ended up being the final crucial puzzle piece. From there, medal hunters set out to find how to input various RGB values in game, but nothing seemed to be working. Red, green, and blue shows up in the video of the map, so was it dying or receiving combo up to those sections chronologically? Could it have been to do with setting certain grades back-to-back? In the default skin, D ranks are red, A ranks are green, and B ranks are blue. Could it have been to do with mods? In the default skin, Hard Rock is red, Easy is green, but wait... Expert Hush-Hush means absolutely no difficulty reducing mods, so that couldn't be it. And while "hues of madness" seemed to be critical in figuring out the medal solution, just what in the world could "Bask in the shade of an ailing mind" mean? Medal hunters thought all of this within the first 2 days. Medal hunters were on the right track so much, that Ephemeral declared that medal hunters would not need a single hint from them, however, with no way to tell if guesses were getting closer or not to the solution, insanity only grew bigger.
```

Comment:
huge block of text it wouldn't hurt to have some double spaces in there.. also concerned about the volume of text overall but at some point you have to say fuck it, especially because such a complicated medal is being covered in depth

also fixed a sentence 
```suggestion
Within the first few minutes of the medal releasing, players immediately noticed both hints and especially the medal icon referenced theyaremanycolors. Ever since the beginning of this medal's release, medal hunters knew the medal involved that map. From there, medal hunters were very quick to piece together that the hues from medal hint/flair are hinting at RGB to some capacity, but nobody knew in what way, and players were less focused on the subhint which ended up being the final crucial puzzle piece.

From there, medal hunters set out to find how to input various RGB values in game, but nothing seemed to be working. Red, green, and blue shows up in the video of the map, so was it dying or receiving combo up to those sections chronologically? Could it have been to do with setting certain grades back-to-back? In the default skin, D ranks are red, A ranks are green, and B ranks are blue. Could it have been to do with mods?

In the default skin, Hard Rock is red, Easy is green, but wait... Expert Hush-Hush means absolutely no difficulty reducing mods, so that couldn't be it. And while "hues of madness" seemed to be critical in figuring out the medal solution, just what in the world could "Bask in the shade of an ailing mind" mean? Medal hunters thought all of this within the first 2 days, and they were on the right track so much, that Ephemeral declared that they would not need a single hint from them. However, with no way to tell if guesses were getting closer or not to the solution, insanity only grew bigger.
```